docs/intro/plans.md

```diff
@@ -44,8 +46,18 @@ the top. Or, if they aren't associated with a test file, put them in a `README.t
 **Any notes about things which are unimplemented must be marked with `TODO:` inside
 the description string.**
 
-## Open a pull request
+## 2. Open a pull request
 
 Open a PR, and work with the reviewer(s) to revise the test plan.
 
 Usually (probably), plans will be landed in separate PRs before test implementations.
+
+## Conventions used in test plans
+
+- `Iff`: If and only if
+- `x=`: cartesian product (for combinatorial test coverage)
```

Comment:
nit: explain that it reads "cartesian-cross equal" a bit like += but with a cartesian product.